### PR TITLE
[Qol][Misc] Make `create-test` script interactive

### PR DESCRIPTION
**Pull request by flx-sta**
## Why am I making these changes?

`npm run create-test` should be intuitive and be able to use without having much knowledge how it works

## What are the changes from a developer perspective?

- Added the `inquirer` module
- They can now simply run `npm run create-test` without any arguments and it will guide them through

### Screenshots/Videos

https://github.com/user-attachments/assets/34de3243-e53e-4523-a78b-f9d94dad9b71

## How to test the changes?

Simply run `npm run create-test`

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: create-test-boilerplate.js [hunk 1]
```diff
@@ -14,59 +15,70 @@ import { fileURLToPath } from 'url';
...
-if (fs.existsSync(filePath)) {
-  console.error(`File "${fileName}.test.ts" already exists.`);
-  process.exit(1);
-}
-
-// Define the content template
-const content = `import { Abilities } from "#enums/abilities";
+const choices = {
+  type: ["Move", "Ability", "Item", "Exit"],
+};
```
Comment (torranx) [inline create-test-boilerplate.js:20]: does this need to be outside the function scope?

Comment (flx-sta) [reply create-test-boilerplate.js]: no. Moved it into the function

**Review by torranx — DISMISSED** [review]
Looks good

**Comment by torranx** [comment]
Added `Lead Dev Review` label because of the package addition

**Review by innerthunder — APPROVED** [review]
lgtm, just depends on whether `inquirer` is wanted

**Comment by flx-sta** [comment]
> lgtm, just depends on whether `inquirer` is wanted

I made sure to put it into dev dependencies so it doesn’t hurt production

**Comment by DayKev** [comment]
image

File: create-test-boilerplate.js [hunk 2]
```diff
@@ -10,63 +6,102 @@ import { fileURLToPath } from 'url';
...
  */
 
+import fs from "fs";
+import inquirer from "inquirer";
+import path from "path";
+import { fileURLToPath } from "url";
+
 // Get the directory name of the current module file
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
```
Comment (DayKev) [inline create-test-boilerplate.js:17]:
```suggestion
const typeChoices = ["Move", "Ability", "Item"];

```

File: create-test-boilerplate.js [hunk 3]
```diff
@@ -10,63 +6,102 @@ import { fileURLToPath } from 'url';
...
+ * Prompts the user to select a type via list.
+ * @returns {Promise<{selectedOption: string}>} the selected type
+ */
+async function promptTestType() {
+  const typeAnswer = await inquirer.prompt([
+    {
+      type: "list",
+      name: "selectedOption",
+      message: "What type of test would you like to create:",
+      choices: ["Move", "Ability", "Item", "EXIT"],
```
Comment (DayKev) [inline create-test-boilerplate.js:28]:
```suggestion
      choices: [...typeChoices, "EXIT"],
```

**Review by innerthunder — APPROVED** [review]

**Review by torranx — APPROVED** [review]

**Review by DayKev — APPROVED** [review]

**Pull request merged**